### ISSUE TITLE
MobileNet does not return activations as commented

### ISSUE THREAD

**Issue by jaxball** [tensorflow/tfjs-core#907]
`models/squeezenet` has a `predictWithActivation` function that returns both named activation and pre-softmax logits. `models/mobilenet` only has a `predict` function which returns a 1D tensor, but its function comment appears identical to that for SqueezeNet, stating that 
>`predict(input: dl.Tensor3D): dl.Tensor1D`
Infer through MobileNet, assumes variables have been loaded. This does standard ImageNet pre-processing before inferring through the model. This method returns named activations as well as pre-softmax logits. 

Can someone verify correctness of the comment and shed light on how to get the activations from MobileNet?

**Comment by dsmilkov** [comment]
This issue was moved to tensorflow/tfjs#64
